### ISSUE TITLE
Accordions are not focusable with the keyboard

### ISSUE THREAD

**Issue by cscuderi** [cehfisher/a11y-style-guide#36]
Accordions (such as the "WCAG 2.0 Guidelines" one on this page: https://cehfisher.github.io/a11y-style-guide/style-guide/section-general.html) are not focusable using just the keyboard. 

In addition, they have no keyboard event listeners (currently only tied to the mouse) and no aria-expanded attributes on them.

**Comment by cehfisher** [comment]
Thanks @cscuderi for catching that bug! It worked at one point, but seems I either never committed that change or it got reverted. I'll work on a fix for that soon.

**Comment by scottaohara** [comment]
to tack onto this issue, be sure to toggle ```aria-expanded="true/false"``` on the hidden content trigger to add context to what the current state of the toggle area is.

**Comment by cehfisher** [comment]
@scottaohara this is the issue i'm hoping you have time to review. if not, send it back my way. thanks!

**Comment by scottaohara** [comment]
yup.  got it

**Comment by cehfisher** [comment]
@cscuderi - thanks to this code from @scottaohara (https://github.com/scottaohara/aria_toggle_blocks) i think the issue you reported has been resolved. can you check out the demo site again? http://a11y-style-guide.com/style-guide/
